kubernetes/5-prometheus.yaml

```diff
@@ -52,12 +52,19 @@ metadata:
 spec:
   groups:
   - name: DemoAlerts
+    # Note: all the rules below need to include the endpoint, namespace, and job
+    # labels to ensure the EDA Pod receives the firing events. Maybe this can be
+    # configured/filtered using some filtering logic on the alertmanager YAMLs?
     rules:
     - alert: InstanceDown 
-      expr: up{job="services"} < 1 
```

Comment:
This alert fired, but never made it to the EDA Pod. I think there's some filtering going on.  Adding the labels to the `expr` fixed it.